src/test/Rets_Api.postman_collection.json

```diff
@@ -194,6 +194,18 @@
 							"disabled": true
 						},
 						{
+
+							"key": "phoneNumbers",
+							"value": "5145556666",
+							"disabled": true
+						},
+						{
+							"key": "phoneNumbers",
+							"value": "5147778888",
+							"disabled": true
+						},
+						{
+
```

Comment:
Please update the request body of "Create Property with all Fields"

Reply:
done in last commit